src/styles/blocks/card.scss

```
@@ -0,0 +1,90 @@
+.card {
+  width: 200px;
+  padding: 15px 14px;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  box-sizing: border-box;
+
+  margin-bottom: 50px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: 15px 5px 41px;
+    transition: transform 1s;
+    border: 0 solid transparent;
+  }
+
+  &__feedback {
+    @include flexSpaceBetween;
+
+    align-items: center;
+    margin-top: 15px;
+    margin-left: 0;
+  }
+
+  &__stars {
+    display: flex;
+    margin: 0;
+  }
+
+  &__price {
+    margin: 20px 0;
+    @include flexSpaceBetween;
+    align-items: baseline;
+  }
+
+  &__button {
+    display: block;
+
+    padding: 11px 10px;
+
+    background-color: #00acdc;
+
+    font-size: 14px;
+    line-height: 16px;
+    color: #fff;
+    text-transform: uppercase;
+    font-weight: bold;
+    text-align: center;
+
+    border-radius: 5px;
+    border: 1px solid transparent;
+
+    transition: color 0.3s, border-color 0.3s, background-color 0.3s;
+    text-decoration: none;
+  }
+
+  &__button:hover {
```

Додай його до кнопки
```suggestion
  &__button:hover {
```